companies/fidelity-de.json

```diff
@@ -0,0 +1,21 @@
+{
+    "slug": "fidelity-de",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "FIL Fondsbank GmbH",
+    "runs": [
+        "Fidelity"
+    ],
+    "address": "Datenschutzbeauftragter\nPostfach 11 06 63\n60041 Frankfurt am Main\nDeutschland",
+    "web": "https://www.fidelity.de",
+    "sources": [
+        "https://www.fidelity.de/datenschutzhinweise/",
+        "https://www.fidelity.de/canonical/1590723880/6574/"
+    ],
+    "suggested-transport-medium": "letter",
```

Comment:
Do you have specific experience with them that would result in this?

Reply:
My reasoning was that they only give a postal address in their privacy document, no email/fax. With the newly added generic email/phone/fax, I will change it to email. I haven't tested it but probably will soon.

Reply:
> My reasoning was that they only give a postal address in their privacy document, no email/fax. 

You don't need to set a `suggested-transport-medium` in that case. Our site will automatically fall back to what's provided in the record.

So, in this case, I think it would be best if we just don't set one.

> I haven't tested it but probably will soon.

It would be great if you could update the record according to #720 after you've done that.

Reply:
Sure, will do!